### PR TITLE
Reworks armor modules minorly, adds lesser versions of baldur/mimir/tyr for free access

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Reworks armor modules and adds lesser versions of some of the armor modules (baldur/try/mimir) for free access by mahrines.
Baldur mk1 gives you better light range at 0.2 slowdown.
Mimir mk1 gives you only the gas permablity effect and comes with a worse version of the helmet that also has that and a gas mask.
Tyr is just old tyr, mark 2 tyr buffed to +15 armor.

## Why It's Good For The Game
Gives modules more use and makes them more balanced overall.

## Changelog
:cl:
add: Added mk1 versions of some of the jaeger modules.
balance: Modules have been rebalanced minorly, added lesser versions of some of the current modules.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -50. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/modular_armor/attachables.dm [hunk 1]
```diff
@@ -42,6 +43,10 @@
 	parent.set_light_range(parent.light_range - power_boost)
 	return ..()
 
+/obj/item/armor_module/attachable/better_shoulder_lamp/mark1
+	name = "\improper Mark 1 Baldur Light Amplification Mark 1 System"
```
Comment (XSlayer300) [inline code/modules/modular_armor/attachables.dm:47]:
```suggestion
	name = "\improper Mark 1 Baldur Light Amplification System"
```

File: code/modules/modular_armor/helmets.dm [hunk 2]
```diff
@@ -41,7 +41,7 @@
 
 
 /obj/item/helmet_module/attachable/mimir_environment_protection
-	name = "Mimir Environmental Helmet System"
+	name = "Mimir Environmental Helmet System Mark 2"
```
Comment (XSlayer300) [inline code/modules/modular_armor/helmets.dm:44]:
```suggestion
	name = "Mark 2 Mimir Environmental Helmet System"
```

File: code/modules/modular_armor/helmets.dm [hunk 3]
```diff
@@ -51,6 +51,13 @@
 	var/permeability_coefficient_mod = -1
 	var/gas_transfer_coefficient_mod = -1
 
+/obj/item/helmet_module/attachable/mimir_environment_protection/mark1 //gas protection
+	name = "Mimir Environmental Helmet System Mark 1"
```
Comment (XSlayer300) [inline code/modules/modular_armor/helmets.dm:55]:
```suggestion
	name = "Mark 1 Mimir Environmental Helmet System"
```

consistency

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 4]
```diff
@@ -1368,6 +1393,14 @@ GLOBAL_LIST_INIT(available_specialist_sets, list("Scout Set", "Sniper Set", "Dem
 		/obj/item/armor_module/armor/legs/marine/eod,
 	)
 
+/obj/effect/essentials_set/mimir
+	desc = "A set of anti-gas gear setup to protect one from gas threats."
+	spawned_gear_list = list(
+		/obj/item/helmet_module/attachable/mimir_environment_protection/mark1,
+		/obj/item/clothing/mask/gas/tactical,
+		/obj/item/armor_module/attachable/mimir_environment_protection/mark1,
+						)
```
Comment (spookydonut) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:1402]: this will fail CI

Comment (Pariah919) [reply code/game/objects/machinery/vending/new_marine_vendors.dm]: i reduced the amount of tabs

**Review by Hughgent — APPROVED** [review]
Nothing immediately strikes me as overly wrong.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-63). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**